Makefile

```diff
@@ -16,11 +16,6 @@ ALL_MODULES := $(shell find . -type f -name "go.mod" -exec dirname {} \; | sort
 
 CMD?=
 
-# TODO: Find a way to configure this in the generated code, currently no effect.
-BUILD_INFO_IMPORT_PATH=go.opentelemetry.io/collector/internal/version
```

Comment:
Do you remember when this package was removed by chance?

Reply:
Looks like it was removed about a year ago https://github.com/open-telemetry/opentelemetry-collector/commit/c5f118ef8667877edc94d87bc033d125674a5b0b